dashboard/app/views/levels/_external.html.haml

```diff
@@ -1,6 +1,7 @@
 %link{rel: 'stylesheet', href: asset_path('blockly/css/common.css')}
 = stylesheet_link_tag 'levels/external', media: 'all'
 = javascript_include_tag 'levels/external'
+-# TODO(dave): Remove options global once all templates consume externalOptions via erb.
```

Comment:
That's maybe a reasonable contract for appOptions (written to by dashboard, consumed by apps).